src/map/entities/charentity.cpp

```diff
@@ -431,17 +431,17 @@ void CCharEntity::setPetZoningInfo()
             {
                 break;
             }
+            petZoningInfo.jugSpawnTime = PPetEntity->getJugSpawnTime();
```

Comment:
good catch. In theory this should be harmless for all pets, but I did add some xi_debug breaks so that was probably triggering it.

Reply:
Yes it was those debug breaks that I moved the jugpet code around for.